### PR TITLE
Gnumakefile doc fixes

### PR DESCRIPTION
**Pull request by jmtd**
Raising this as a separate PR because I'm not able to properly test it myself.

### CODE REVIEW

**Comment by chungy** [comment]
at least the win32 makefile works properly.

File: pkg/win32/GNUmakefile [hunk 1]
```diff
@@ -56,6 +56,8 @@ staging-%:
 
 	$(TOPLEVEL)/man/simplecpp -D_WIN32 -DPRECOMPILED  \
 	             -D$(shell echo $* | tr a-z A-Z)      \
+	             -DLONG_GAME_NAME="$(PACKAGE_NAME)"   \
```
Comment (fragglet) [inline pkg/win32/GNUmakefile]: This rule is used to build staging directories for every game, so I think this means you'll be saying "Chocolate Doom", "chocolate-doom", in cases where you mean "Chocolate Heretic", "chocolate-hexen", etc.?

Comment (jmtd) [reply pkg/win32/GNUmakefile]: Yes; I'd Mia interpreted what this was used for; I thought it was a top level README in a package of all engines. 

I'm lost in a twisty maze of passages, all alike!

**Comment by chungy** [comment]
Is this PR still relevant? As far as win32 is concerned, there's no issue with the simplecpp invocation in master.

**Comment by jmtd** [comment]
> Is this PR still relevant? As far as win32 is concerned, there's no issue with the simplecpp invocation in master.

I can't remember. I'll try to look again this week.

**Comment by jmtd** [comment]
> I can't remember. I'll try to look again this week.

I didn't but might as well close this unless someone notices a problem.

**Pull request closed without merging**